process/block/metablock.go

```diff
@@ -146,7 +146,8 @@ func (mp *metaProcessor) ProcessBlock(
 				"for shard", headerHandler.GetShardID(),
 			)
 
-			go mp.requestHandler.RequestMetaHeader(headerHandler.GetPrevHash())
+			// TODO : check epoch
```

Comment:
if headerHandler.IsStartOfEpoch than epoch - 1, otherwise headerHandler.GetEpoch.

Reply:
not relevant anymore